src/main/java/persistence/sql/ddl/AnnotationUtils.java

```diff
@@ -0,0 +1,12 @@
+package persistence.sql.ddl;
+
+public class AnnotationUtils {
+    public static <T> Object getDefaultValue(Class<T> clazz, String name) {
```

Comment:
```suggestion
    public static <T> Object getMethodDefaultValue(Class<T> clazz, String methodName) {
```
로 좀 더 구체적으로 표현해볼 수 있겠네요 :)

Reply:
오.. 저는 AnnotationUtils에서 기본 값을 가져온다. 이렇게 까지만 추상화/캡슐화 해도 괜찮다고 생각했습니다.